### ISSUE TITLE
ReactPDF.renderToStream Not working when files are larger than 13MB

### ISSUE THREAD

**Issue by airidasj** [diegomura/react-pdf#2055]
**Describe the bug**
RenderToStream / renderToString / renderToString does not Work with bigger fileSize.

**To Reproduce**
In our case we make an invoice for products and we include more images of the product (4) to a single pdf page. When we render 2-3 products, pdf renders with no issues. 

When we add 4-10 products, and the image files increase PDF file size, the renderToStream function stops, and never produces a result.

We can render the pdf on the browser successfully, no matter the filezsize. Doesnt work in Node API.

**Desktop (please complete the following information):**
 - OS: [e.g. MacOS M1]
- Trying to render in Backend (Node Js server environment)
 - React-pdf version [e.g. v3.0.0]

**Comment by airidasj** [comment]
The function does not produce any errors. It just hangs with no output.

**Comment by anwarjaved** [comment]
Yes, I am facing the same issue with generating pdf with lots of images

**Comment by chathu-novade** [comment]
Can you try running on a Docker container with platform `platform=linux/amd64` ?

**Comment by airidasj** [comment]
Tested on platform=linux/amd64.

Works fine on Linux. 

Seems to be an issue on Apple M1 (and Windows, regarding previous comment).

**Comment by chathu-novade** [comment]
@airidasj I think it's may be due to some dependency is not supports M1 (arm64) yet. We got issues with node-canvas not working on M1.

**Comment by diegomura** [comment]
How can I replicate this?

**Comment by anwarjaved** [comment]
> How can I replicate this?

Just need to render lots of images like 10-12 of 1MB size or more
